src/app/wallets/update-on-chain-receipt.ts

```diff
@@ -129,3 +144,69 @@ const processTxForWallet = async (
     }
   })
 }
+
+const processTxForHotWallet = async ({
+  tx,
+  logger,
+}: {
+  tx: IncomingOnChainTransaction
+  logger: Logger
+}): Promise<void | ApplicationError> => {
+  const coldStorageService = await ColdStorageService()
+  if (coldStorageService instanceof Error) return coldStorageService
+
+  const isFromColdStorage = await coldStorageService.isWithdrawalTransaction(
+    tx.rawTx.txHash,
+  )
+  if (coldStorageService instanceof Error) return coldStorageService
```

Comment:
This is duplicate of line 156 you probably mean `isFromColdStorage` here or?

Reply:
Fixed https://github.com/GaloyMoney/galoy/pull/986/commits/6ab22bcee786a64aab9e81490eb1ce77f907d9ce